Addon/MultiColumnContentAreaRenderer.cs

```diff
@@ -101,16 +105,21 @@ public void RenderContentAreaItemsInternal(IHtmlHelper htmlHelper, IEnumerable<C
                 var contextResult = currentContext.RenderItem(htmlHelper, current,
                     () =>
                     {
+                        var tag = GetContentAreaItemTemplateTag(htmlHelper, current);
```

Comment:
I think it would be good to have theses assigned before RenderContentAreaItem is called so the metadata is available there

Reply:
I no longer define the metadata within that scope. I do it earlier in the for loop